plugins/TodayAtMun/DiaryParser.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+from os import path
+from pathlib import Path
+
+import certifi
+import urllib3
+from bs4 import BeautifulSoup
+
+
+class DiaryParser:
+    DATA_SOURCE = "https://www.mun.ca/regoff/calendar/sectionNo=GENINFO-0086"
+
+    def __init__(self):
+        self.diary = {}
+        http = urllib3.PoolManager(cert_reqs="CERT_REQUIRED", ca_certs=certifi.where())
+        mun_request = http.request("GET", self.DATA_SOURCE)
+        soup = BeautifulSoup(mun_request.data, "html.parser")
+        left_aligned_data = soup.find_all("td", attrs={"align": "left"})
+        right_aligned_data = soup.find_all("td", attrs={"align": "justify"})
```

Comment:
What is the left aligned data and the right aligned data?

Reply:
Renamed to clarify